pyedflib/_extensions/_pyedflib.pyx

```diff
@@ -615,10 +620,14 @@ def rewind(handle, edfsignal):
     """void edfrewind(int handle, int edfsignal)"""
     c_edf.edfrewind(handle, edfsignal)
 
+def set_sex(handle, sex):
+    """int edf_set_sex(int handle, int sex)"""
+    if sex is None: return 0 #don't set sex at all to prevent default 'F'
+    return c_edf.edf_set_gender(handle, sex)
+
 def set_gender(handle, gender):
-    """int edf_set_gender(int handle, int gender)"""
-    if gender is None: return 0 #don't set gender at all to prevent default 'F'
-    return c_edf.edf_set_gender(handle, gender)
+    warnings.warn('This function is deprecated.', DeprecationWarning, stacklevel=2)
+    set_sex(handle, gender)
```

Comment:
```suggestion
    return set_sex(handle, gender)
```

Reply:
hat error is gone in the new version written from scratch grin